falcon_apispec/falcon_plugin.py

```diff
@@ -35,12 +36,15 @@ def path_helper(self, operations, resource, **kwargs):
         operations.update(yaml_utils.load_operations_from_docstring(resource.__doc__) or {})
         path = resource_uri_mapping[resource]
 
+        if base_path is not None:
+            base_path = '/' + base_path.strip('/')
+            path = re.sub(base_path, "", path, 1)
```

Comment:
This isn't clear without having to think about it... what exactly is going on?

Maybe a comment might help?